.github/styles/Vocab/Products/accept.txt

```diff
@@ -47,49 +84,112 @@ dev
 Dexalot
 Docusaurus
 dotenv
+drizzleOptions
 ecrecover
+Enums
+env
 EOAs
+ERC
+ERC721s
 ESLint
+Eteng
 Ethereum
+Ethers
+ethtx
+eVoting
+expectRevert
 feeRecipient
+finney
+fuji
+Fuji
+gamification
+Ganache
+getOwner
+getters
+gif
 GitHub
 github
 Golang
 gwei
 Gwei
+Hardforks
+Harpalsinh
+hdwallet
+HDWallet
+i.e.
 incentivized
+Infura
+ipfs
+IPFS
 IPs
+isRegistered
+Jadeja
 JavaScript
+jpg
 json
 JSON
+Keccak
 keyName
+keystore
 KYCed
 liveness
+lol
 lookups
+mainnet
```

Comment:
Same with Fuji, mainnet should have first letter upper-case.

Reply:
Resolved